### PR TITLE
Unique link text

### LINKED ISSUE #319: link to pube does not have descriptive text `find out more` -> `visit whitepube site` or similar

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #319

## Description

Have put in the changed link text for project "visit site" link

@geeksforsocialchange/developers

### CODE REVIEW

File: themes/gfsc/layouts/index.html.html [hunk 1]
```diff
@@ -56,8 +56,14 @@ <h3 class="window__header">{{ .Title }}</h3>
...
               <div class="window__footer">
                 <div class="window__footer__buttonbox">
-                  <a class="window__footer__button btn" href="{{ .Permalink }}"
-                    >Browse</a
+                  <a
+                    class="window__footer__button btn"
+                    href="{{ .Permalink }}"
+                    aria-label="Browse {{ .Title }} projects"
+                    >Browse
+                    <span class="screen-reader-only"
```
Comment (r-ferrier) [inline themes/gfsc/layouts/index.html.html:64]: I think you need to pick one way of doing this or the other - I think the screenreader will ignore the screenreader only class here in favour of the aria-label. `Browse about {{.Title}}` doesn't quite make sense to me?

**Comment by aaaaargZombies** [comment]
I see that kim has suggested to add `aria-label` to avoid having lots of differently sized buttons but the original issue is about project pages `find out more` not telling being informative enough. This isn't really an issue with screenreaders, you should be able to tell where your going if you see a link. This came up from the starling checklist we use.

> We give links descriptive names, so you know exactly what they’re about. You won’t find any ‘click heres’ in these parts.

Example in the issue would be to switch to something project specific `find out more` -> `visit whitepube site` or similar.

**Review by r-ferrier — CHANGES_REQUESTED** [review]
This was a good audit but I think some of these should maybe stay as they are? I can see an argument for adding a project name to a 'find out more' button at the bottom of a project page, because you might have skipped down there, but I think for the browse buttons on the filters, for example, it actually makes more sense as is? The problem with an isolated "click here" link is usually that it's sitting in a big block of text and the screenreader has to read all of it for it to make any sense. I've recorded a couple of videos so you can see how it is using the screenreader with them.

You've also used screenreader-only text on some of these which should be removed if you're using an `aria-label`. I *think* it's preferred to change the text itself where possible but with buttons there's a visual consideration too so I think it's the right use for `aria-label`

https://github.com/geeksforsocialchange/gfsc-v3/assets/46496391/0d777484-e2ed-4d5d-bf60-cd6bb61b0690



https://github.com/geeksforsocialchange/gfsc-v3/assets/46496391/af9911d4-fdda-4651-bbf8-3754d8bf2210

**Comment by r-ferrier** [comment]
sorry, reading back I'm not sure I made that very clear - I think the project index change is good, just needs the span class removing, and maybe as @aaaaargZombies says, get some feedback from content team, and revert the other changes to repeat buttons in lists

File: themes/gfsc/layouts/index.html.html (hunk 1 above)
Comment (ivan-kocienski-gfsc) [reply themes/gfsc/layouts/index.html.html]: I have removed all the hidden text / aria labels

**Comment by r-ferrier** [comment]
From a code perspective this looks much better, but I'm really not sure about the content - it just doesn't always make sense to say this for every project. eg `Visit The White Pube Website Overhaul site ` makes no sense and `Visit Hulme Community Garden Centre website consultancy site` just takes you to a trello board of user stories.

I'm sorry I didn't properly look at all use cases for this before the last review, I assumed because you'd taken this approach the buttons were all taking you to a linked site and the projects were all named after said sites. I think probably best for now to close this and have a rethink with the content team if we want to address this in the future because it probably needs more project specific wording, as @aaaaargZombies says above?

**Comment by ivan-kocienski-gfsc** [comment]
Closing this as the AC could be better defined.

What we need to do is to go through the site looking at all the places where links/buttons have repetitive texts and make a case-by-case decision on how that text can be extended into sentences that make grammatical sense. This may involve either changing the content (i.e. the title of a project) or it may involve putting in a new field in the markdown that can have the specific text.

**Pull request closed without merging**